support/src/main/java/org/jenkinsci/plugins/workflow/support/steps/build/RunWrapper.java

```diff
@@ -170,4 +172,14 @@ public String getId() throws AbortException {
         }
     }
 
+    @Whitelisted
+    public @Nonnull String getAbsoluteUrl() throws AbortException {
+        String root = Jenkins.getActiveInstance().getRootUrl();
+        if (root == null) {
+            throw new AbortException("Jenkins root url is not configured!");
+        } else {
+            return root + build().getUrl();
```

Comment:
:ant: this is slightly different from `Run.getAbsoluteUrl`. The handling of `Stapler.getCurrentRequest()` probably does not matter, because that would be null from the Workflow CPS VM thread, but the failure to call `Util.encode` might matter. Better to just use the standard method here.